NavigationFlowCoordinator/NavigationFlowCoordinator.swift

```diff
@@ -165,7 +176,9 @@ open class NavigationFlowCoordinator: FlowCoordinator {
     ///   - animated: is transition animated
     ///   - completion: closure called when transition is finished
     public func present(viewController: UIViewController, animated: Bool = true, completion: (() -> Void)? = nil) {
-        navigationController.present(viewController, animated: true, completion: completion)
+        let nvc = UINavigationController(rootViewController: viewController)    // add standard UINVC bar
```

Comment:
Internally wrapping new VC in UINavigationController might be confusing. Man can expect that pushing new VC using current coordinator api will use UINavigationController that wraps mainVC. It won't, it will use UINavigationController below our current modal VC.
What's more we might want to present main VC without nav bar. At the moment we would have to hide nav bar explicitly after starting coordinator.

Reply:
You're correct about the NavBar showing.  The user would have to manually hide it in their VC.  Unless we add another parameter for the coordinator to control it.